### DISCUSSION TITLE
Can we have Custom email IMAP Setup

### DISCUSSION THREAD

**Discussion by bhavya3024** [zoidsh/meru#366] [Feature Requests]
I have a different email domain name to setup on gmail dekstop, How can we enable it ?
In Android and IOS version of gmail app, this feature is available already.

**Comment by JJWatMyself** [comment]
Using the app as an email client is slightly different to using the app to access Gmail web.  For the Android Gmail, published by Google, the app dose indeed have an additional feature to be an email client. 

Something to consider is using the server based email client feature.  Settings, All Settings, Accounts, Add a mail account.  You can also create Filters to add Labels to tag the messages sent to each account. 

![image](https://github.com/timche/gmail-desktop/assets/22590445/54d8d5f0-2585-41e3-99a3-1892b66bf62c)
![image](https://github.com/timche/gmail-desktop/assets/22590445/2f3e902b-963f-4c08-b29b-0f85891f82ce)
![image](https://github.com/timche/gmail-desktop/assets/22590445/2c22462c-a79e-4de8-a034-b8d93d0760de)

![image](https://github.com/timche/gmail-desktop/assets/22590445/7e2a6c1e-7461-475a-82b3-91fb5007f8da)
